autogpt_platform/backend/backend/blocks/search.py

```diff
@@ -180,3 +190,82 @@ def run(self, input_data: Input, **kwargs) -> BlockOutput:
             yield "condition", weather_data["weather"][0]["description"]
         else:
             raise RuntimeError(f"Expected keys not found in response: {weather_data}")
+
+
+class FactCheckerBlock(Block, GetRequest):
```

Comment:
move to /jina folder and reuse the _auth.py there

Reply:
will do :)